front/src/components/map/filter/PeriodFilterButton.vue

```diff
@@ -106,10 +106,8 @@ export default {
       this.filterPosts();
     },
     async filterPosts() {
-      this.$store.commit("post/CLEAR_POSTS");
-      const filteredPosts = await this.$store.dispatch("filter/filterPosts");
-      this.$store.commit("post/SET_POSTS", filteredPosts);
-      if (filteredPosts.length === 0) {
+      await this.$store.dispatch("post/loadPosts");
+      if (this.$store.getters["post/posts"].length === 0) {
```

Comment:
posts.length === 0인지 검사하는 로직이 다른 곳에서도 쓰이고 있는데, 해당 로직을 처리하는 메소드를 getters 내부에 두고, getter를 호출하는 방식이 좋을 것 같네요.

```suggestion
      if (this.$store.getters["post/isEmpty"]) {
```